src/PDL.cxx

```diff
@@ -84,6 +85,7 @@ ParticleTable read_pdl_file(const std::string& filename)
     ParticleTable particleFactory;
     std::copy(PDLIterator(inFile), PDLIterator::end(),
               inserter(particleFactory));
+    set_parities(particleFactory);
```

Comment:
`set_parties` is a function created for the particular `.pdl` file in our data directory. We should not use it in the main `yap` code.